src/main/java/frc/robot/subsystems/drive/SwerveModuleIOTalonFXSim.java

```diff
@@ -127,29 +132,35 @@ public void config() {
         //  which means that TorqueCurrent.PeakForwardTorqueCurrent and related won't affect it
         final InvertedValue driveInvertedValue = InvertedValue.CounterClockwise_Positive;
         driveTalonFXConfiguration.Slot0 = new Slot0Configs()
-                .withKV(0.973);
-        driveTalonFXConfiguration.TorqueCurrent.PeakForwardTorqueCurrent = 60;
-        driveTalonFXConfiguration.TorqueCurrent.PeakReverseTorqueCurrent = -60;
+                .withKP(50)
+                .withKS(4.796)
+                .withKA(2.549);
```

Comment:
So are these FF and PID gains appropriate for simulation? If they haven't been tuned specifically for simulation, they should be, as currently, the system dynamics differ between the simulated swerve and the real robot swerve.

Reply:
These are from master. They seem to work.